client/components/planning-editor-standalone/profile-fields.ts

```diff
@@ -35,6 +35,14 @@ export const getPlanningProfileFields = (): Array<IFieldConverted> => {
     for (const fieldId of planningFieldIds) {
         const fieldSchema = planningProfile.schema[fieldId];
 
+        /**
+         * If a field does not have show_in_embedded_editor or required toggled on
+         * we must not show it in the embedded editor
+         */
+        if (!(fieldSchema.show_in_embedded_editor || fieldSchema.required)) {
```

Comment:
We'll eventually use `getPlanningProfileFields` for non-embedded editors too. Add a argument which would tell if you want to get fields for embedded editor or not.